lib/Swagger/src/TemplateClassification/Storage/Api/TCSApi.php

```diff
@@ -238,31 +236,27 @@ public function getTemplateTypeWithHttpInfo($wiki_id, $page_id)
         if ($page_id === null) {
             throw new \InvalidArgumentException('Missing the required parameter $page_id when calling getTemplateType');
         }
-  
         // parse inputs
         $resourcePath = "/{wiki_id}/{page_id}";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "/{wiki_id}/{page_id}" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)